runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -230,6 +230,25 @@ public void removeNode(@Nonnull Layout currentLayout,
         reconfigureSequencerServers(currentLayout, newLayout, false);
     }
 
+    /**
+     * Attempts to force comment a new layout to the cluster.
+     *
+     * @param currentLayout the current layout
+     * @param forceLayout the new layout to force
+     * @throws QuorumUnreachableException
+     */
+    public void forceLayout(@Nonnull Layout currentLayout, @Nonnull Layout forceLayout)
+            throws QuorumUnreachableException {
+        currentLayout.setRuntime(runtime);
+        sealEpoch(currentLayout);
```

Comment:
I do not think this will work when we have multiple failures and cluster is in minority. Also you do not need to do this here. You can do it the way committed message does.

Reply:
I try to seal and ignore the exception. This way I at least try to seal the servers that are alive.